lib/archivist/index.js

```diff
@@ -25,6 +25,10 @@ exports.setup = function (cb) {
 	configuration.setup(logger, cb);
 };
 
+exports.connectVaults = function (cb) {
```

Comment:
We should also have disconnectVaults

Reply:
As we have closeVaults, I will name it openVaults